notebooks/official/model_monitoring/get_started_with_model_monitoring_custom.ipynb

```diff
@@ -962,17 +939,17 @@
       "source": [
         "### Create the monitoring job\n",
         "\n",
-        "You create a monitoring job, with your monitoring specifications, using the `aiplatform.ModelDeploymentMonitoringJob.create()` method, with the following parameters:\n",
+        "Create a monitoring job, with your monitoring specifications, using Vertex AI's `ModelDeploymentMonitoringJob.create()` method, with the following parameters:\n",
```

Comment:
perhaps add a link to the `ModelDeploymentMonitoringJob.create()` method:

https://cloud.google.com/python/docs/reference/aiplatform/1.48.0/summary_method#google_cloud_aiplatform_ModelDeploymentMonitoringJob_create_summary